Gulpfile.ts

```diff
@@ -153,7 +156,7 @@ const librarySourceMap = [
 
     // JavaScript + all host library
     { target: "lib.d.ts", sources: ["header.d.ts", "es5.d.ts"].concat(hostsLibrarySources) },
-    { target: "lib.es6.d.ts", sources: ["header.d.ts", "es5.d.ts"].concat(es2015LibrarySources, hostsLibrarySources, "dom.iterable.d.ts") }
+    { target: "lib.es6.d.ts", sources: ["header.d.ts", "es5.d.ts"].concat(es2015LibrarySources, hostsLibrarySources, es2015HostLibrarySources) }

```

Comment:
you also need to add entries in [commandlineparser.ts](https://github.com/Microsoft/TypeScript/blob/0b125a356e88679974912ac5b3ead0b1634fd9d1/src/compiler/commandLineParser.ts#L411-L438) for the two new lib targets.